android/build.gradle

```diff
@@ -93,6 +95,13 @@ dependencies {
     testImplementation "org.assertj:assertj-core:$assertj_version"
 
     testImplementation "io.mockk:mockk:$mockk_version"
+    // Core library
+    testImplementation 'junit:junit:4.12'
+    testImplementation 'androidx.test:core:1.2.0'
+    // AndroidJUnitRunner and JUnit Rules
+//    testImplementation 'org.robolectric:robolectric:4.3.1'
+    testImplementation 'androidx.test.ext:junit:1.1.1'
+    testImplementation 'org.robolectric:robolectric:4.3.1'
```

Comment:
had to make this changes because I added a call to `Log.e`, which is part of the Android dependencies and the `android.jar` that is used with Unit tests don't have implementations so we need to use roboelectric, which has default implementations for all these calls.